packages/tocco-ui/src/Table/Table.js

```diff
@@ -0,0 +1,221 @@
+import React, {useState, useRef, useEffect} from 'react'
+import PropTypes from 'prop-types'
+import {FormattedMessage} from 'react-intl'
+
+import StyledTable, {
+  PaginationContainer,
+  StretchingTableContainer, StyledDnD, StyledFullRow, StyledFullRowProgress, StyledTableBody,
+  StyledTableHead, StyledTableHeaderCell, StyledTableRow,
+  StyledTableWrapper
+} from './StyledTable'
+import SortingState from './SortingState'
+import ResizingController from './ResizingController'
+import {LoadingSpinner, Pagination, Typography} from '../'
+import useResize from './useResize'
+import useDnD from './useDnD'
+import useSelection from './selection/useSelection'
+import StaticCell from './StaticCell'
+import {selectionStylePropType} from './selection/selectionStyles'
+import {getSelectionCell} from './selection/selectionColumnEnhancer'
+import {columnPropType, dataPropType, keyPropType} from './propTypes'
+
+const InProgressRow = () => <StyledTableRow>
+  <StyledFullRowProgress>
+    <LoadingSpinner size="20"/>
+    <Typography.P>
+      <FormattedMessage id="client.component.table.dataLoading"/>
+    </Typography.P>
+  </StyledFullRowProgress>
+</StyledTableRow>
+
+const NoDataRow = () => <StyledTableRow>
+  <StyledFullRow>
+    <Typography.Span>
+      <FormattedMessage id="client.component.table.noData"/>
+    </Typography.Span>
+  </StyledFullRow>
+</StyledTableRow>
+
+const ThContent = ({column, data}) =>
+  column.HeaderRenderer
+    ? <column.HeaderRenderer column={column} data={data} />
+    : <div dangerouslySetInnerHTML={{__html: column.label}}/> || null
```

Comment:
what is the purpose of `|| null` at the end? will that ever be used? won't that `<div>` after the `:` always be rendered if no `HeaderRenderer` is set?